### ISSUE TITLE
Dynamic Background Getting Split

### ISSUE THREAD

**Issue by SasiruVirajith** [surfbryce/beautiful-lyrics#838]
### How did you install Beautiful Lyrics?

Marketplace

### What is the Bug/Issue?

![Screenshot 2024-08-14 093121](https://github.com/user-attachments/assets/d28bed8c-399d-4415-9b67-19b5ebcf7e7b)
![Screenshot 2024-08-14 093217](https://github.com/user-attachments/assets/295a1fd2-2c68-4661-837a-64a03cd2ddee)
![Screenshot 2024-08-14 093242](https://github.com/user-attachments/assets/338cb06e-cded-4fea-be38-61a131408fa7)

So as you can see the issue is the Dynamic Background being split and it's annoying me a lot. I tried many things but nothing was fixed. However, the issue will be solved if I uninstall Spotify and reinstall it. But that's not a proper solution, right? Even though I reinstall Spotify, the same issue will come back right after a new Spotify update comes in. Btw, I'm not using the Microsoft Store version. So please I'm asking for a proper solution. Also, I'm not sure whether this has something to do with the Experimental Features as well. So I've attached a screenshot of the search results of the 'sidebar' in the Experimental Features.

![Screenshot 2024-08-14 093629](https://github.com/user-attachments/assets/72e8fa98-a617-4b12-936c-bef3e184d70c)
![Screenshot 2024-08-14 093659](https://github.com/user-attachments/assets/b5928512-2456-43ad-bd1b-f7a9e40d8fd3)

In case there is no solution, I'd be delighted to use Beautiful Lyrics with no dynamic background. Also following are the only things which I've installed through the Spicetify Marketplace

![image](https://github.com/user-attachments/assets/eb84c1f6-1ecb-4195-9d0c-347c33fc1f4a)

### What Country are you from?

Sri Lanka

### What Operating System are you using?

Windows

### Spotify/Spicetify Information

Spotify Version: 1.2.44.405.g81fd6352
Spotify Runtime: cef_125.0.19+g3d8f1c9+chromium-125.0.6422.112
Spicetify Version: 2.37.1
Spicetify Theme: Onepunch / dark
Spicetify Extensions: [adblock.js, beautiful-lyrics.mjs]
Spicetify Custom Apps: [marketplace]

### Spotify HTML/CSS and Console Screenshots

![image](https://github.com/user-attachments/assets/4ee6a54d-a837-446b-b6a7-3ce5e2f6af7e)

[Spotify HTML and CSS files.zip](https://github.com/user-attachments/files/16607437/Spotify.HTML.and.CSS.files.zip)

### Information Checklist

- [X] Spotify/Spicetify Information
- [X] Spotify HTML File
- [X] Spotify CSS File
- [X] Screenshots of Developer Tools Console Tab

**Comment by RandomXYZ46** [comment]
Have you found the answer? I also had the same issue.

**Comment by re1e4se** [comment]
I also ran into this issue and found a temporary fix by modifying the CSS.

What I did:
```css
.main-nowPlayingView-contextItemInfo:before,
.E08D6ucrHuPJYzzGO7HG:before,
.E08D6ucrHuPJYzzGO7HG:after {
  background: none !important
}
```

**Note:** If you have canvas experiment (`Enables short, looping visuals on tracks.`) enabled, either with `canvas-play-loop` or `canvas-play-on-hover`, the lyrics on the sidebar may look different than what it was looking like before version `1.2.44.405.g81fd6352` since the darkening effect will not be present due to the issue and this temporary fix.

**Steps on how to apply the fix**:
1. Go to Marketplace.
2. Navigate to Snippets.
3. Click `Add CSS` in the bottom right corner.
4. Copy & Paste the CSS snippet to `Custom CSS`.
5. Give it a name.
6. Click Save CSS.
